src/app/containers/FrontPageMain/index.jsx

```diff
@@ -32,7 +35,13 @@ const FrontPageMain = ({ frontPageData }) => {
     <>
       <ATIAnalytics data={frontPageData} />
       <ChartbeatAnalytics data={frontPageData} />
-      <MetadataContainer metadata={metadata} promo={promo} />
+      <MetadataContainer
```

Comment:
I really like how this refactor has made it very clear what values in the Medatadata and LinkedData vary per pagetype. 💯